strictEqual.js

```diff
@@ -0,0 +1,11 @@
+const strictEqual = (a, b) => {
```

Comment:
Naming note, single letter variables are avoided everywhere with the exception of `i` in loops. They don't provide any context. `val1` and ``val2` would be better here. Definitely heading into the realm of nitpicky but worth being aware of.